app/controllers/trainees/placements_controller.rb

```diff
@@ -59,6 +75,23 @@ def destroy
       redirect_to(trainee_placements_confirm_path(trainee_id: @trainee.slug))
     end
 
+    def search
```

Comment:
We decided to avoid doing custom actions in controller early on in the project in order to keep them as restful as possible. I think it would be good to maintain this consistency as i think we've done a pretty good job so far. In the publish PR i think we just had a simple search controller which  had `new` for displaying the search form and `create` for creating a search.

Reply:
Good policy. I had moved these to `PlacementSchoolSearchesController` as RESTful actions but forgot to remove this old code.